pom.xml

```diff
@@ -113,10 +113,11 @@
     </distributionManagement>
 
     <properties>
-        <java.version>1.8</java.version>
+        <java.version>8</java.version>
         <maven.version>3.9.9</maven.version>
-        <maven.compiler.source>${java.version}</maven.compiler.source>
-        <maven.compiler.target>${java.version}</maven.compiler.target>
+        <!-- Pinning compiler target to Java 8-->
+        <maven.compiler.target>8</maven.compiler.target>
```

Comment:
IIRC，In principle, `maven.compiler.source` and  `maven.compiler.target` should be set as a pair.